coremltools/converters/mil/frontend/torch/torch_op_registry.py

```diff
@@ -127,3 +129,14 @@ def func_wrapper(func):
         # decorator called without argument
         return func_wrapper
     return func_wrapper(_func)
+
+
+def is_torch_fx_node_supported(torch_fx_node: torch.fx.Node) -> bool:
+    if torch_fx_node.op != "call_function":
+        return False
+
+    torch_fx_node_target_name = torch_fx_node.target.__name__.lower()
+    if torch_fx_node_target_name.startswith("aten."):
+        torch_fx_node_target_name = torch_fx_node_target_name[5:]
```

Comment:
This `5` looks a little bit random, although I know you are trying to remove the `aten.` prefix.

One better solution is to have a variable such as 
```
_ATEN_NODE_PREFIX = "aten."  # I just randomly come up a variable name here, feel free to change.
...
torch_fx_node_target_name = torch_fx_node_target_name[len(_ATEN_NODE_PREFIX):]
```

Reply:
Revised as suggested